AndroidAnnotations/androidannotations-core/androidannotations-api/src/main/java/org/androidannotations/annotations/PageScrolled.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (C) 2010-2015 eBusiness Information, Excilys Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed To in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.androidannotations.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * <p>
+ * This annotation is intended to be used on methods to receive events defined
+ * by
+ * <code>android.support.v4.view.ViewPager.OnPageChangeListener.OnPageScrolled</code>
+ * when the current page is scrolled, either as part of a programmatically
+ * initiated smooth scroll or a user initiated touch scroll.
+ * </p>
+ * <p>
+ * The annotation value should be one or several R.id.* fields that refers to
+ * ViewPager or subclasses of ViewPager. If not set, the method name will be
+ * used as the R.id.* field name.
+ * </p>
+ * <p>
+ * The method MAY have multiple parameter :
```

Comment:
We should declare that they are all optional, but the order should must be this.